app/views/shared/_email_languages.html.erb

```diff
@@ -10,13 +10,7 @@ locals:
       label: t('forms.registration.labels.email_language'),
       hint: local_assigns.fetch(
         :hint,
-        t(
-          'account.email_language.languages_list',
-          app_name: APP_NAME,
-          list: I18n.available_locales.
-            map { |locale| t("account.email_language.name.#{locale}") }.
-            to_sentence(last_word_connector: " #{t('account.email_language.sentence_connector')} "),
```

Comment:
`account.email_language. sentence_connector` key is unused now, right?

Reply:
Appears so! Let's see what the build thinks of it, but I'll remove after verifying.

Reply:
It was caught by the tests 🙏  Removed in 46fdcaf02. Thanks for highlighting.